autoscheduler/frontend/assets/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this for PWA-ification of the website?

Reply:
Yeah I think so - was just one of the things generated from realfavicongenerator.net. Believe this one is for like if you saved it on windows somehow, it'd be the icon that shows as the tile.